src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("niteris");
+        user.setPassword("qwerty123");
+        user.setAge(18);
+    }
+
+    @Test
+    void register_NullLogin_NotOK() {
```

Comment:
in camel case, the first letter after "_ " must be in lower case. Example: register_nullAge_notOk. You should pay attention to the name "_NotOK". Please looking all tests names.

Example from Google Java Style Guide:
Prose form | Correct | Incorrect
"new customer ID" | newCustomerId | newCustomerID